docs/security.md

```diff
@@ -93,12 +93,13 @@ options are available to modify the confinement:
   `default`. When specified without `caps`, `caps` defaults to being empty. Not
   compatible with `security-override` or `security-policy`.
 * `security-override`: (optional) high level overrides to use when
-  `security-template` and `caps` are not sufficient - see
-  [advanced usage](https://wiki.ubuntu.com/SecurityTeam/Specifications/SnappyConfinement)
-  for details. Not compatible with `caps`, `security-template` or
-  `security-policy`
-    * `apparmor: path/to/security.override`
-    * `seccomp: path/to/filter.override`
+  `security-template` and `caps` are not sufficient. Not compatible with
+   `caps`, `security-template` or `security-policy`
+  The following keys are supported:
+    * `read-paths`: a list of additional paths that the app can read
+    * `write-paths`: a list of additional paths that the app can write
+    * `abstractions`: a list of additional abstractions for the app
+    * `syscalls`: a list of additional syscalls that the app can use
```

Comment:
This is an area I really hope we can clean up with capabilities, but I'm not yet sure we'll have time for 16.04. It feels like way too many variations for something that should be internal to the system, and offered via an abstraction that we own.